inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/metric/phase/ReadPhaseMetricData.java

```diff
@@ -0,0 +1,105 @@
+/*
+ *   Licensed to the Apache Software Foundation (ASF) under one
+ *   or more contributor license agreements.  See the NOTICE file
+ *   distributed with this work for additional information
+ *   regarding copyright ownership.  The ASF licenses this file
+ *   to you under the Apache License, Version 2.0 (the
+ *   "License"); you may not use this file except in compliance
+ *   with the License.  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.apache.inlong.sort.base.metric.phase;
+
+import static org.apache.inlong.sort.base.Constants.READ_PHASE_TIMESTAMP;
+
+import java.util.Map;
+import org.apache.flink.metrics.Counter;
+import org.apache.flink.metrics.MetricGroup;
+import org.apache.inlong.sort.base.metric.MetricData;
+import org.apache.inlong.sort.base.metric.MetricOption;
+import org.apache.inlong.sort.base.metric.ThreadSafeCounter;
+
+/**
+ * A collection class for handling read phase metric data
+ */
+public class ReadPhaseMetricData implements MetricData {
+
+    private final MetricGroup metricGroup;
+    private Counter readPhase;
+    private final Map<String, String> labels;
+
+    public ReadPhaseMetricData(MetricOption option, MetricGroup metricGroup) {
+        this.metricGroup = metricGroup;
+        this.labels = option.getLabels();
+        ThreadSafeCounter readPhaseCounter = new ThreadSafeCounter();
+        readPhaseCounter.inc(option.getReadPhase());
+        registerMetricsForReadPhase(readPhaseCounter);
+    }
+
+    /**
+     * User can use custom counter that extends from {@link Counter}
```

Comment:
the code comment is suitable for common situation, here needs a doc more specific

Reply:
> the code comment is suitable for common situation, here needs a doc more specific

okay,I will change the description.